wallet/reservation.c

```diff
@@ -56,6 +57,7 @@ static void reserve_and_report(struct json_stream *response,
 			       u32 reserve,
 			       struct utxo **utxos)
 {
+	char *utxo_string = tal_strdup(NULL, "");
```

Comment:
```suggestion
	char *utxo_string = "";
```

Is is not the same? but without duplication?

Reply:
The string needs to be tal allocated so it can be appended to